src/meta/proto-conv/src/util.rs

```diff
@@ -81,6 +81,10 @@ const META_CHANGE_LOG: &[(u64, &str)] = &[
         "2023-01-07: Add: new-schema pb::DataType to/from TableDataType",
     ),
     (25, "2023-01-05: Add: user.proto/OnErrorMode::AbortNum"),
+    (
+        26,
+        "2023-01-16: Add: metadata.proto/DataSchema::max_column_id",
+    ),
```

Comment:
You'll have to add a test case to ensure the new version works:

https://github.com/datafuselabs/databend/blob/78d127257519bea475ad1361ee0359da764a443b/src/meta/proto-conv/tests/it/v024_table_meta.rs